include/highfive/H5PropertyList.hpp

```diff
@@ -194,6 +194,70 @@ class MetadataBlockSize {
 };
 
 ///
+/// \brief Configure the file space strategy.
+///
+/// See the upstream documentation of `H5Pget_file_space_strategy` for more details. Essentially,
+/// it enables configuring how space is allocate in the file.
+///
+class FileSpaceStrategy {
+  public:
+    ///
+    /// \brief Create a file space strategy property.
+    ///
+    /// \param strategy  The HDF5 free space strategy.
```

Comment:
```suggestion
    /// \param strategy The HDF5 free space strategy.
```
Whitespace, please.

Reply:
Still there? Is this from formatting? The line does not align, either. I'm puzzled.